pkg/autoscaler/autoscaler/query/external.go

```diff
@@ -1,113 +0,0 @@
-// Copyright 2020 PingCAP, Inc.
```

Comment:
We still support external, keep this file or you want to do it all over again?

Reply:
i will keep this file